src/Schema/SchemaInterface.php

```diff
@@ -206,6 +206,10 @@ interface SchemaInterface extends ConstraintSchemaInterface
      * Define the abstract column type as `binary`.
      */
     public const TYPE_BINARY = 'binary';
+    /**
+     * Define the abstract column type as `bit`.
+     */
+    public const TYPE_BIT = 'bit';
```

Comment:
As it is not a global type that behaves the same for all dbms, i think it would be a good idea to map it in db-pgsql, and each dbms will have its own behavior, as it should be.

Reply:
`public const TYPE_JSONB = 'json';` is also for `db-pgsql` only.
Should we also move it into `Yiisoft\Db\Pgsql\Schema`?

Reply:
I think so, basically we should have mapped in db, only the generic types, and the types with specific behaviors in each implementation.